vocab/types.json

```diff
@@ -1148,15 +1148,15 @@
         "translatableTo": null
     },
     "https://openminds.ebrains.eu/core/GRIDID": {
-        "description": null,
+        "description": "A GRID (Global Research Identifier Database) identifier.",
```

Comment:
I suggest to follow the same logic as in "Structured information about a digital object identifier, as standardized by the International Organization for Standardization."

Reply:
Almost the only information in any digital identifier schema is the actual identifier (there's the pattern, but that's not a property), so I don't think it makes sense to say "Structured information about..."